packages/design-system/src/foundation/colors/index.ts

```diff
@@ -54,11 +153,63 @@ const createCSSVariables = () => {
   return cssVars;
 };
 
+const createTokenColorVariables = (surface: string) => {
+  const cssVars: { [name: string]: string } = {};
+  for (let coreToken in token.tokenCoreColor) {
+    if (token.tokenCoreColor.hasOwnProperty(coreToken)) {
+      const varName = `--${coreToken}`;
+      const tokenValue =
+        token.tokenCoreColor[coreToken as keyof typeof token.tokenCoreColor];
+      cssVars[varName] = `var(${
+        tokenValue[surface as keyof typeof tokenValue]
+      })`;
+    }
+  }
+
+  for (let componentGroup in token.tokenComponentColor) {
```

Comment:
```suggestion
  let componentGroup: keyof typeof token.tokenComponentColor;
  for (componentGroup in token.tokenComponentColor) {
```
이런식으로 하면 as를 쓸 필요가 없는지 확인 가능하실까요? 
에픽리엑트 269p~270p를 참고했습니다.

Reply:
355f34fe51bbaa5e89a27efb9612d93e3a9c476e
특별한 type 없이 정의했던 token color 값들에 대해 interface 를 새로 정의하고, 해당 interface 를 코드내에 사용하도록 변경하여 as 를 이용해 타입 단언 하는 부분을 삭제했습니다. 확인 부탁드립니다!

Reply:
아 그리고 에픽리액트 아니고 이펙티브 타입스크립트(소곤소곤)